s3_to_redshift.go

```diff
@@ -26,6 +27,7 @@ var (
 	dataDate        = flag.String("date", "", "data date we should process, must be full RFC3339")
 	configFile      = flag.String("config", "", "schema & table config to use in YAML format")
 	gzip            = flag.Bool("gzip", true, "whether target files are gzipped, defaults to true")
+	timeGranularity = flag.String("granularity", "day", "how often we expect to append new data")
```

Comment:
Even in your PR you mention explicitly referencing a day in the payload. Feeling like if we're doing that anyways, let's be strict about requiring you pass in the granularity.